client/packages/system/src/Immunisation/api/hooks/useImmunisationProgram.ts

```diff
@@ -126,10 +149,10 @@ const useCreate = () => {
 
 const useUpdate = (
   id: string,
-  setErrorMessage: Dispatch<SetStateAction<string>>
+  setErrorMessage: Dispatch<SetStateAction<string>>,
+  t: TypedTFunction<LocaleKey>
 ) => {
   const { api, storeId, queryClient } = useImmunisationGraphQL();
-  const t = useTranslation('catalogue');
```

Comment:
Weird one - it was the fact that there is this useTranslation call here that the AppBar was moving down when you clicked opened the create modal... passing in instead 🤷‍♀️

Reply:
I don't like passing around the t function - though for no good reason, just has worked out to be clumsy when I've done it, requiring a lot of parameters. I tested with removing the passed in function and the issue doesn't happen for me now ( I did have it initially and after further testing.. it's all fine :shrug: )

Reply:
The plot thickens! 😅

Reply:
Hm, I'm still getting it consistently after a page refresh - I agree the passing around is a bit unfortunate... and I don't get it, I swear we conditionally render components with that hook in them all the time??

Will leave for now, maybe future us will have more insight 🤞